src/main/java/no/nav/familie/integrasjoner/arbeidsfordeling/ArbeidsfordelingRestClient.kt

```diff
@@ -31,6 +32,23 @@ class ArbeidsfordelingRestClient(@Value("\${NORG2_URL}")
                                     .toUri())
     }
 
+    fun finnBehandlendeEnhetMedBesteMatch(gjeldendeTema: no.nav.familie.kontrakter.felles.Tema,
+                                          gjeldendeGeografiskOmråde: String?,
+                                          gjeldendeDiskresjonskode: String?,
+                                          erEgenAnsatt: Boolean): List<Enhet> {
+        return postForEntity(
+                UriComponentsBuilder.fromUri(norg2Uri)
+                        .pathSegment("api/v1/arbeidsfordeling/enheter/bestmatch")
+                        .build()
+                        .toUri(),
+                ArbeidsfordelingKritierie(diskresjonskode = gjeldendeDiskresjonskode,
+                                          geografiskOmraade = gjeldendeGeografiskOmråde,
+                                          tema = gjeldendeTema.toString(),
```

Comment:
Burde kanskje bruke name her? (enum)
```suggestion
                                          tema = gjeldendeTema.name,
```